### PR TITLE
rename and align config inputs

### PR DESCRIPTION
**Pull request by crazy-max**
follow-up https://github.com/docker/buildx/pull/2268

rename
* `config` to `buildkitd-config`
* `config-inline` to `buildkitd-config-inline`

to be aligned with https://github.com/docker/buildx/pull/2268 and deprecate old inputs as well (will be removed on next major)

@dvdksn needs docs follow-up in https://docs.docker.com/build/ci/github-actions/configure-builder/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,23 +85,23 @@ The following inputs can be used as `step.with` keys:
...
+| `platforms`                | List/CSV |                    | Fixed [platforms](https://docs.docker.com/engine/reference/commandline/buildx_create/#platform) for current node. If not empty, values take priority over the detected ones. |
+| `append`                   | YAML     |                    | [Append additional nodes](docs/advanced/append-nodes.md) to the builder                                                                                                      |
+| `cache-binary`             | Bool     | `true`             | Cache buildx binary to GitHub Actions cache backend                                                                                                                          |
+| `cleanup`                  | Bool     | `true`             | Cleanup temp files and remove builder at the end of a job                                                                                                                    |
+
+> * ¹ `buildkitd-config` and `buildkitd-config-inline` are mutually exclusive
```
Comment (dvdksn) [inline README.md:104]: GFM supports footnotes so we can use this syntax: 
https://gist.github.com/dvdksn/7ebcc3839ca66b25ee66c453c404cdb7

Comment (crazy-max) [reply README.md]: Arg it puts the footnote after contributing section 😢 

https://github.com/crazy-max/docker-setup-buildx-action/tree/fix-inputs

**Review by dvdksn — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (dvdksn) [reply README.md]: ah ok. Yeah I guess that makes sense in a way... "footnotes" 😆 If you don't want that you could just do something like

```
| foo  | bar |
|------|-----|
| a \* |     |

_\* lorem ipsum_
```

| foo  | bar |
|------|-----|
| a \* |     |

_\* lorem ipsum_

**Pull request merged**